### DISCUSSION TITLE
Convert 22.5 kHz to 44.1 kHz

### DISCUSSION THREAD

**Discussion by TedBias** [audiamus/AaxAudioConverter#83] [General]
Is it possible to convert the audio file from 22.5 kHz to 44.1 kHz when doing a conversion from AAX to M4a?
I have a few audiobooks that are the lower sample rate and they do not play properly on my set up.  If I could change the sample rate when I convert, that would solve my playback issue.

**Comment by audiamus** [comment]
Theoretically yes, but upscaling is not implemented. And that was by intention. Some users believe upscaling improves audio quality, but it doesn't, of course. It just makes the files bigger.

Are you really sure, it's the bandwidth/bit rate that is causing your problems? Have you tried MP3? Or even M4B? Some players treat M4A and M4B differently although it's exactly the same binary data.

**Comment by LeeBinder** [comment]
native Audible aax uses 44.1 kHz sample rate inside their m4b!
